Add computed data to VDC storage profile data source (#782)
vmware/terraform-provider-vcd: website/docs/d/storage_profile.html.markdown

* `name` - (Required) Storage profile name.

## Attribute reference
* `id` - storage profile identifier

(Supported in provider *v3.6+*)

> * `limit` - Maximum number of storage bytes (scaled by Units) allocated for this profile. A value of 0 is understood to mean `maximum possible`

Reviewer: ```suggestion
* `limit` - Maximum number of storage bytes (scaled by 'units' field) allocated for this profile. A value of 0 is understood to mean `maximum possible`
```

Author: updated

> * `used_storage` - Storage used, in Megabytes, by the storage profile

Reviewer: Maybe it is just me, but it sounds strange - could this be simpler.

```suggestion
* `used_storage` - Storage used, by the storage profile (in Megabytes)
```

Author: changed

> * `default` - True if this is default storage profile for this vDC. The default storage profile is used when an object that can specify a storage profile is created with no storage profile specified

Reviewer: ```suggestion
* `default` - True if this is default storage profile for this VDC. The default storage profile is used when an object that can specify a storage profile is created with no storage profile specified
```

* `enabled` - True if this storage profile is enabled for use in the vDC"
* `iops_allocated` - Total IOPS currently allocated to this storage profile
* `units` - Scale used to define Limit
* `iops_settings` - A block providing iops settings. See [IOPS settings](#iopsSettings) below for details.

<a id="iopsSettings"></a>
## IOPS settings

(Supported from VCD *v10.2*)

> * `iops_limiting_enabled` - True if this storage profile is IOPs-based placement enabled

Reviewer: ```suggestion
* `iops_limiting_enabled` - True if this storage profile is IOPS-based placement enabled
```

* `maximum_disk_iops` - The maximum IOPS value that this storage profile is permitted to deliver. Value of 0 means this max setting is disabled and there is no max disk IOPS restriction

> * `default_disk_iops` - Value of 0 for disk iops means that no iops would be reserved or provisioned for that virtual disk

Reviewer: ```suggestion
* `default_disk_iops` - Value of 0 for disk iops means that no IOPS would be reserved or provisioned for that virtual disk
```

Author: fixed

> * `disk_iops_per_gb_max` - The maximum disk IOPs per GB value that this storage profile is permitted to deliver. A value of 0 means there is no perGB IOPS restriction

Reviewer: ```suggestion
* `disk_iops_per_gb_max` - The maximum disk IOPs per GB value that this storage profile is permitted to deliver. A value of 0 means there is no per GB IOPS restriction
```

Author: fixed

> * `iops_limit` - Maximum number of IOPs that can be allocated for this profile. A value of 0 is understood to mean `maximum possible`

Reviewer: Maybe a simplified name

```suggestion
* `iops_limit` - Maximum number of IOPs that can be allocated for this profile. `0` means `maximum possible`
```
